### DISCUSSION TITLE
Map with all Photos -> 500 HttpException Allowed memory size of 268435456 bytes exhausted (tried to allocate 20975616 bytes)

### DISCUSSION THREAD

**Discussion by DKeppi** [LycheeOrg/Lychee#2473] [Bugs]
### Lychee version

v5.4.0

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

When i try to open the Map to see all my photos on it ~ 11000, i get an Error:

500
HttpException
Allowed memory size of 268435456 bytes exhausted (tried to allocate 20975616 bytes)

### Steps to reproduce the issue

Click on Map Button on the Main page with all albums on it

### Diagnostics [REQUIRED]

  Warning: Dropbox import not working. dropbox_key is empty.
        Info: Latest version of PHP is 8.3
        Warning: Online updates are disabled by configuration

### Browser & System [REQUIRED]

Chrome
Lychee Docker Container on Synology DS920+

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
Well loading 11 000 pictures is pushing it a bit. 🥲
There might also be a memory leak. 😔

**Reply by DKeppi** [reply]
I understand, so no chance to see the map for the whole library! Ok, i will use it only for the album 👍🏻😎
